### ISSUE TITLE
How do I download it??

### ISSUE THREAD

**Issue by Mkcein** [MIvanchev/HitmanC47TimingFix#4]
I couldn't download it, it's giving me an error, can you tell me where I can download this file for the STEAM version?

**Comment by MIvanchev** [comment]
Use this link for the steam version: https://ci.appveyor.com/api/buildjobs/ihhqrtd2kf9lyoc0/artifacts/patcher-other.exe. Sorry, the links died because I haven't uploaded them to a permanent location. I'm currently busy on another project but I'll be back on this soon :)

**Comment by Mkcein** [comment]
> 
This message is appearing, can you make the file available on some mega download / mediafire, etc...?
{"message":"Artifact not found or access denied."}

**Comment by MIvanchev** [comment]
I'm so stupid, I sent the wrong link again: https://ci.appveyor.com/api/buildjobs/l35sw5u151vw66gr/artifacts/patcher-other.exe is the current build.

>can you make the file available on some mega download / mediafire, etc...?

I can, just need some free time to do it all :D

Take care, tell me how it went, I don't have experience with Steam.

**Comment by Mkcein** [comment]
Thank you for your attention in advance, however the patch did not work, it displays this error message when generating the ZIP file, I used administrator mode, the zip file is created but the content is inaccessible.

Error photo link:
https://imgur.com/a/u9vQcjj

**Comment by MIvanchev** [comment]
That's odd... you are placing it and running it in the game dir right?

**Comment by Mkcein** [comment]
yes in the root directory of the game, I'm using windows 10, I used administrator privileges, it even tries to create the ZIP file but an error occurs.

**Comment by MIvanchev** [comment]
OK, try to delete the zip file and try again. I'm looking in the source where it might occur.

**Comment by Mkcein** [comment]
The same problem occurred again, I uninstalled Hitman and installed again his path is at:

C:\Program Files (x86)\Steam\steamapps\common\Hitman Codename 47

**Comment by MIvanchev** [comment]
OK, let me try something. I think this bug is easy to fix. If not we call it a day :)

**Comment by MIvanchev** [comment]
Try this: https://ci.appveyor.com/api/buildjobs/gdfweebvxfqtheiw/artifacts/patcher-other.exe

Thanks a lot for your patience, it's very important for us open source people!

P.S.
Delete the ZIP file first!!!

**Comment by junkandshit** [comment]
i compiled both versions here is working binaries https://mega.nz/folder/tqMETbJC#5By1avHfo1JH_6EzE0TSzw

**Comment by MIvanchev** [comment]
@junkandshit Cool, I think I have found some hosting after all, we'll be on it with some more fixes starting next month I hope. Did the patch work for you?

**Comment by junkandshit** [comment]
at 1024x768 resolution i am getting 60 FPS, but at 1920x1080 there is no frame limit, i am searching for full hd fix. My setup ryzen/radeon/win10, radeon chill or radeon frame rate target control not working..

**Comment by MIvanchev** [comment]
Yeah, I have some initial full HD work going on, it's hard because of GUI rescaling for all platforms, stay tuned :)

**Comment by VicCoca** [comment]
Hi, is this mod no longer available?

**Comment by MIvanchev** [comment]
Just a sec.

**Comment by MIvanchev** [comment]
@VicCoca 

https://ivanchev.net/files/public/patcher-other.exe
https://ivanchev.net/files/public/patcher-gog.exe
